perceval/algorithm/sampler.py

```diff
@@ -193,7 +209,7 @@ def n_iterations(self):
     def _probs_wrapper(self, progress_callback: callable = None):
         # max_shots is used as the invert of the precision set in the probs computation
         # Rationale: mimic the fact that the more shots, the more accurate probability distributions are.
-        precision = None if self._max_shots is None else min(1e-6, 1/self._max_shots)
+        precision = None if self._max_shots is None else min(1e-6, 1 / self._max_shots)
```

Comment:
Typing error in L217 inside method `_samples_wrapper`, change simumation -> simulation.